src/backend/base/langflow/components/embeddings/AstraVectorize.py

```diff
@@ -50,37 +50,38 @@ class AstraVectorize(Component):
     inputs = [
         DropdownInput(
             name="provider",
-            display_name="Provider name",
+            display_name="Provider",
             options=VECTORIZE_PROVIDERS_MAPPING.keys(),
             value="",
+            required=True,
         ),
         MessageTextInput(
             name="model_name",
-            display_name="Model name",
+            display_name="Model Name",
             info=f"The embedding model to use for the selected provider. Each provider has a different set of models "
-            f"available (full list at https://docs.datastax.com/en/astra-db-serverless/databases/embedding-generation.html):\n\n{VECTORIZE_MODELS_STR}",
+            f"available (https://docs.datastax.com/en/astra-db-serverless/databases/embedding-generation.html):\n\n{VECTORIZE_MODELS_STR}",
             required=True,
         ),
+        SecretStrInput(
+            name="provider_api_key",
+            display_name="Provider API Key",
+            info="An alternative to the Astra Authentication that passes an API key for the provider with each request to Astra DB. This may be used when Vectorize is configured for the collection, but no corresponding provider secret is stored within Astra's key management system.",
+        ),
         MessageTextInput(
             name="api_key_name",
-            display_name="API Key name",
+            display_name="Provider API Key Name",
             info="The name of the embeddings provider API key stored on Astra. If set, it will override the 'ProviderKey' in the authentication parameters.",
+            advanced=True,
```

Comment:
Is there a common case where the `providerKey` is set differently than the default?

Reply:
what do you mean? there's no default value. The value is defined by the user

Reply:
This is the key name for the authentication token, correct? e.g.
```
CollectionVectorServiceOptions(
           ...
            authentication={
                "providerKey": os.environ["OPENAI_API_KEY"],
           }
        )
```

But going through the UI flow, I see that you are correct in your other comment that the user-defined key name is required during creation. So I will undo these changes, and have `Provider API Key Name` shown by default, and `Provider API Key` in Advanced.